ychat/src/commonMain/kotlin/co/yml/ychat/YChat.kt

```diff
@@ -93,6 +94,21 @@ interface YChat {
      */
     fun imageGenerations(): ImageGenerations
 
+    /**
+     * The edits api is used to edit prompts and re-generate. Given a prompt and an instruction,
+     * the model will return an edited version of the prompt.
+     *
+     * You can configure the parameters of the edits before executing it. Example:
+     * ```
+     * val result = YChat.create(apiKey).edits()
```

Comment:
I think the snippet of code in the Javadoc might be wrong. The instruction we are passing in the execute method. I Think It should be:

   * ```
     * val result = YChat.create(apiKey).edits()
     *      .setInput("As Descartes said, I think, therefore")
     *      .setResults(1)
     *      .set...
     *      .execute("Fix spelling mistakes")
     * ```